web/src/routes/OrganizationsList/Organization/Tabs/TeamsAndMembership/TeamsView/ManageMembers/AddNewTeamMemberDrawer.tsx

```diff
@@ -0,0 +1,194 @@
+import {
+  ActionGroup,
+  Button,
+  Divider,
+  DrawerActions,
+  DrawerCloseButton,
+  DrawerHead,
+  DrawerPanelBody,
+  DrawerPanelContent,
+  Form,
+  FormGroup,
+  Spinner,
+} from '@patternfly/react-core';
+import {SelectOption, SelectGroup} from '@patternfly/react-core/deprecated';
```

Comment:
This needs to be updated so both of these components come from `'@patternfly/react-core` instead now.